consensus/src/consensus_observer/network_message.rs

```diff
@@ -140,16 +143,10 @@ impl ConsensusObserverDirectSend {
     pub fn get_content(&self) -> String {
         match self {
             ConsensusObserverDirectSend::OrderedBlock(ordered_block) => {
-                format!(
-                    "OrderedBlock: {}",
-                    ordered_block.ordered_proof.commit_info()
-                )
+                format!("OrderedBlock: {}", ordered_block.proof_block_info())
             },
             ConsensusObserverDirectSend::CommitDecision(commit_decision) => {
-                format!(
-                    "CommitDecision: {}",
-                    commit_decision.ledger_info().commit_info()
-                )
+                format!("CommitDecision: {}", commit_decision.proof_block_info())
             },
             ConsensusObserverDirectSend::BlockPayload(block_payload) => {
                 format!(
```

Comment:
Isn't this a `Display` impl?

Reply:
Yeah, this is used by the `Display` impl 😄 Did you want me to inline it? I have a PR which makes use of it in other places, e.g., logs. But, I can inline it (and update those) if you prefer.

Reply:
Yeah, that might be cleaner.

Reply:
Sounds good. Will follow up in a future PR 😄